ArduCopter/mode.h

```diff
@@ -1138,3 +1138,49 @@ class ModeAvoidADSB : public ModeGuided {
 private:
 
 };
+
+class ModeChase : public ModeGuided {
+
+public:
+
+    ModeChase(Copter &copter) :
+        Copter::ModeGuided(copter) {
+        target_loc.flags.relative_alt = true;
+    }
+
+    bool init(bool ignore_checks) override;
+    void run() override;
+
+    bool requires_GPS() const override { return true; }
+    bool has_manual_throttle() const override { return false; }
+    bool allows_arming(bool from_gcs) const override { return false; }
+    bool is_autopilot() const override { return true; }
+
+    bool set_velocity(const Vector3f& velocity_neu);
+
+    void mavlink_packet_received(const mavlink_message_t &msg);
+
+protected:
+
+    const char *name() const override { return "CHASE"; }
+    const char *name4() const override { return "CHSE"; }
```

Comment:
I think it would be nice if there is a rule of 4 character description of flight mode.
Is "ATUN" AutoTUNe or AuTotUNe? "BRAK" is "BRAKe".